### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by sim-codes**
#### Update Readme #99
I updated the README and added CONTRUBTING file for contributors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,41 +9,50 @@ AIxplora is your new personal assistant, an open-source project that opens up un
...
-
-- Documentation: Help us make our project more understandable and user-friendly.
-
-- Testing: Every bug found is a step towards perfection.
-
-- Suggest Features: We believe in the power of ideas, no matter where they come from.
-
-- Spread the Word: Share our project within your networks. The more people know about AIxplora, the better it can become!
+## Contributing
+We welcome contributors of all backgrounds and interests, testing, accessibility, community building, and writing code. If you would like to contribute code, feel free to check out our  [contributing guide](https://github.com/grumpyp/aixplora/blob/main/CONTRIBUTING.md).
```
Comment (grumpyp) [inline README.md:16]: **nitpick:** will this link work?

**Comment by grumpyp** [comment]
Thanks for your contribution. Would you additionally extend the README with a bit more information about our features and our roadmap? That would be awesome.

File: README.md (hunk 1 above)
Comment (sim-codes) [reply README.md:16]: Sure, but it is broken currently, I will fix it.

**Comment by sim-codes** [comment]
Sure I would love to.
Kindly assigned it to me. Thanks

File: README.md [hunk 2]
```diff
@@ -9,41 +9,50 @@ AIxplora is your new personal assistant, an open-source project that opens up un
...
 
 ## Demo and introduction videos
-[![Summary function showcase](https://img.youtube.com/vi/8x9HhWjjNtY/hqdefault.jpg)](https://youtu.be/8x9HhWjjNtY)
-https://youtu.be/2lNNKLM0o7U
-https://youtu.be/eKLmhJobVvc
+- [Summarize documents](https://youtu.be/8x9HhWjjNtY)
+- [Audio query](https://youtu.be/2lNNKLM0o7U)
+- [Vector databases and LLMs query](https://youtu.be/eKLmhJobVvc)
```
Comment (grumpyp) [inline README.md:22]: theres a new video here:
https://youtu.be/JW2mX0aJl2U

would you add it please?

**Review by grumpyp — COMMENTED** [review]
left some comment. 

would be awesome if we could highlight the tool itself with its features in the beginning, and that we are looking for contributors and partners/investors to speed up the development too,..

**Comment by sim-codes** [comment]
I will check and do the needful.

**Comment by grumpyp** [comment]
done in d1a4382eaa779f722115f9da89b654f2e8f06370

**Pull request closed without merging**